bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableHBaseVeneerSettings.java

```diff
@@ -779,8 +778,9 @@ private ClientOperationTimeouts buildCallSettings() {
             Optional.<Duration>absent(),
             extractUnaryAttemptTimeout(BIGTABLE_MUTATE_RPC_ATTEMPT_TIMEOUT_MS_KEY),
             extractOverallTimeout(
-                BIGTABLE_MUTATE_RPC_TIMEOUT_MS_KEY,
-                BigtableOptionsFactory.BIGTABLE_LONG_RPC_TIMEOUT_MS_KEY));
+                    BIGTABLE_MUTATE_RPC_TIMEOUT_MS_KEY,
+                    BigtableOptionsFactory.BIGTABLE_LONG_RPC_TIMEOUT_MS_KEY)
+                .or(bulkMutateDefaultTimeout()));
```

Comment:
I think that USE_BATCH default should override the deprecated BIGTABLE_LONG_RPC_TIMEOUT_MS_KEY.

Something like this:

```java
// The default overall timeout for BulkMutate when USE_BATCH is enabled
DEFAULT_BATCH_BULK_MUTATE_OVERAL_TIMEOUT = Duration.ofMinutes(20);

bulkMutateOverallTimeout = configuration.getBoolean(BIGTABLE_USE_BATCH, false) ?
    extractOverallTimeout(BIGTABLE_MUTATE_RPC_TIMEOUT_MS_KEY).or(Optional.of(DEFAULT_BATCH_BULK_MUTATE_OVERAL_TIMEOUT)
   : extractOverallTimeout(BIGTABLE_MUTATE_RPC_TIMEOUT_MS_KEY, BIGTABLE_LONG_RPC_TIMEOUT_MS_KEY));
```

Reply:
Sure, updated!